src/components/layout/Footer.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components/macro'
+
+const Footer = styled.footer`
+  height: 50px;
```

Comment:
this could also be indicated in the MainWrapper

Reply:
It would not be visible in styleguidist, when there is no height declared at this point. I have tried it. :D ... I do not know, if there is a better way?